src/main/groovy/org/pih/warehouse/core/ReceivingBinSearchCommand.groovy

```diff
@@ -0,0 +1,17 @@
+package org.pih.warehouse.core
+
+/**
+ * Used to filter the returned results when listing/searching internal receiving bin locations.
+ */
+class ReceivingBinSearchCommand extends InternalLocationSearchCommand {
```

Comment:
I know the term "receiving bin" has been used a lot throughout the system and maybe in requirements, but I would love to get rid of the notion of a "receiving bin" once and for all. It's more appropriate to call these locations "receiving locations". It's usually not a bin in the typical sense of a "storage bin" or "bin location". It's a staging location on the floor of the warehouse near the dock door.

Reply:
![image](https://github.com/user-attachments/assets/7b411256-4030-4ba5-8406-2b7dac6a1710)